### ISSUE TITLE
[Layout] - Discussion about design

### ISSUE THREAD

**Issue by kazzkiq** [caravel-tool/caravel-tool.github.io#1]
Here is what I've been thinking for a landing-page (WIP):

![layout](https://cloud.githubusercontent.com/assets/1953194/14737841/94a5e76c-0854-11e6-9953-3a11058a8e1b.png)

The idea would be to give a simple few-steps tutorial, then link the user to other pages containing Documentation and more complex examples.

What do you think, @baltazarparra ?

**Comment by baltazarparra** [comment]
Very nice, useful and simple. can i change some design elements? and i like to see another deploy page for content reference. Up .psd, please.

**Comment by kazzkiq** [comment]
Sure, link to the .psd: https://www.dropbox.com/s/c4pl63fjr6kwjvs/layout.psd?dl=0

And the discussion about the Caravel-status layout: https://github.com/caravel-tool/caravel-status/issues/1

**Comment by baltazarparra** [comment]
thax

**Comment by kazzkiq** [comment]
@baltazarparra Any news about the layout? 😋

**Comment by baltazarparra** [comment]
Sorry for delay... This week, promise. @kazzkiq My job is killing me

**Comment by kazzkiq** [comment]
No problem, I just asked because at the end of the week there will be probably an alpha version ready for tests, so it would be cool to start working on the website, etc. :)

**Comment by baltazarparra** [comment]
i will change some elements tomorrow, but there is alpha version layout too, kkk

![caravel](https://cloud.githubusercontent.com/assets/7395304/14980372/66c3d7de-10fe-11e6-935a-e6f643663dbf.png)

**Comment by kazzkiq** [comment]
@baltazarparra Sorry for the late response, it was really hard for me to work on the project these last months.

I liked the clean style from the proposal. My only request would be to remove the internationalization option as, for now, there is no intent into translating the website/docs to other languages (we will keep it in English). Otherwise, you can definitely keep going with this path 👍 

I started to work on the project again and next week there will be a "stable" release of Caravel. I'm willing to publish the website and basic documentation by then, do you think you can build a "final" version of the layout until next weekend? Are you still able to help in the project?

**Comment by baltazarparra** [comment]
Dont worry dude, im on ;) 

i do also front too?

**Comment by kazzkiq** [comment]
Awesome!

I'm going to configure the project structure so we can work on it once the design is finished. [Qualy boilerplate](https://github.com/Qualy-org/qualy-front) will be used for this task (without Jade).

**Comment by baltazarparra** [comment]
@kazzkiq project structure its ok?

**Comment by kazzkiq** [comment]
@baltazarparra Yes. I've upgraded to [Brunch](http://brunch.io/) instead of Qualy. The project will be using **SASS** for styles and **Vanilla JavaScript** for logic (if needed).

Here is how you can run it at your machine:
1. Clone repo;
2. Run `npm install`;
3. Run `npm install -g brunch`;
4. Run `brunch watch --server`;

Now you can access the project at **`localhost:3333`** and have live updates on code change.

All the dev code is put into `/app` folder (there are some samples there already), while the generated code is automatically created at the `/public` folder.

You can create `.scss` files inside `/app/styles` folder and they will be automatically compiled.

If you have any questions, please let me know.

**Comment by baltazarparra** [comment]
I have a short time available, so I'll split the process. @KarolinedeAlencar She will participate in this stage of development.

**Comment by kazzkiq** [comment]
@baltazarparra @KarolinedeAlencar Perfect. We can keep updating this issue if there are any questions and/or progress to be shown.
